### PR TITLE
Add mailchimp embed form

### PR DESCRIPTION
**Pull request by ganeshbsub**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,24 +17,16 @@
...
-            <form class="register-form">
+            <form class="register-form" action="//simplabs.us13.list-manage.com/subscribe/post?u=50bd3378ab1da4988aa174d0b&amp;id=fe31a777a8" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
               <div class="form-group">
                 <div class="input-group">
-                  <input type="email" name="email" class="form-control"/>
+                  <input type="email" name="EMAIL" id="mce-EMAIL" class="form-control"/>
+                  
                   <div class="input-group-addon">
-                    <button type="submit">Sign Up</button>
+                    <button type="submit" id="mc-embedded-subscribe">Register</button>
```
Comment (marcoow) [inline index.html]: This should stay "Sign Up"

**Comment by marcoow** [comment]
also please rebase this on the `gh-pages` branch - seems outdated

**Comment by ganeshbsub** [comment]
Weird. I checked commit history. It is up to date with gh-pages branch.

**Comment by marcoow** [comment]
ah sorry, guess I didn't look close enough. Please squash commits though

**Pull request merged**